packages/react-core/src/components/CodeBlock/examples/CodeBlockExpandable.tsx

```diff
@@ -85,11 +86,17 @@ url: https://raw.githubusercontent.com/Azure-Samples/helm-charts/master/docs`;
     <CodeBlock actions={actions}>
       <CodeBlockCode>
         {code}
-        <ExpandableSection isExpanded={isExpanded} isDetached contentId="code-block-expand">
+        <ExpandableSection isExpanded={isExpanded} isDetached contentId="code-block-expand" toggleId={toggleId}>
           {expandedCode}
         </ExpandableSection>
       </CodeBlockCode>
-      <ExpandableSectionToggle isExpanded={isExpanded} onToggle={onToggle} contentId="code-block-expand" direction="up">
+      <ExpandableSectionToggle
+        isExpanded={isExpanded}
+        onToggle={onToggle}
+        contentId="code-block-expand"
+        direction="up"
+        toggleId={toggleId}
+      >
         {isExpanded ? 'Show Less' : 'Show More'}
```

Comment:
Since we're here, let's just update this toggle text similarly to your ExpandableSection PR, "Show less expandable example content" and "Show more..."